src/utilops.h

```diff
@@ -44,10 +44,12 @@ GenericDialog *file_util_gen_dlg(const gchar *title,
 				 const gchar *role,
 				 GtkWidget *parent, gint auto_close,
 				 void (*cancel_cb)(GenericDialog *, gpointer), gpointer data);
-FileDialog *file_util_file_dlg(const gchar *title,
+GtkWidget *file_util_file_dlg(const gchar *title,
 			       const gchar *role,
 			       GtkWidget *parent,
-			       void (*cancel_cb)(FileDialog *, gpointer), gpointer data);
+			       void (*cancel_cb)(FileDialog *, gpointer),
```

Comment:
`FileDialog` and related functions are not used anymore and could be removed.

Also whole `ui-pathsel` module becomes unused.